### PR TITLE
Fix/embedded nature popup

### PR DESCRIPTION
**Pull request by mikkoisa**
- Fix embedded nature area view missing popup
- Ad  left side map focus border to views without sidebar

### CODE REVIEW

File: src/views/MapView/MapView.js [hunk 1]
```diff
@@ -137,6 +137,7 @@ const MapView = (props) => {
   };
 
   const setClickCoordinates = (ev) => {
+    if (embeded) return;
```
Comment (mhieta) [inline src/views/MapView/MapView.js:140]: Should this constrant be `embedded ` ?

**Review by mhieta — CHANGES_REQUESTED** [review]
Please check/fix constant naming, otherwise good 👍

File: src/views/MapView/MapView.js (hunk 1 above)
Comment (mikkoisa) [reply src/views/MapView/MapView.js:140]: True, will change those!

**Review by mhieta — APPROVED** [review]
Good 👍

**Pull request merged**